pennylane/transforms/tape_expand.py

```diff
@@ -467,7 +467,7 @@ def circuit():
     Now let's set up a context where the custom decomposition will be applied:
 
     >>> with qml.transforms.set_decomposition({qml.CNOT : custom_cnot}, dev):
-    ...     print(qml.draw(circuit)())
+    ...     print(qml.draw(circuit, wire_order=[0, 1])())
```

Comment:
with DQL, this stayed 0-1 regardless. with new DQ, it swaps them because `1` comes first in the tape. regression in drawing behaviour?

Reply:
We can update `drawer/draw.py:L262` to be:
```
            _wire_order = wire_order or qnode.device.wires or qnode.tape.wires
```

Which I think would actually be the better behavior.  Want me to open up a quick fix for that?

Reply:
sure!